packages/seal-meilisearch-adapter/src/MeilisearchSearcher.php

```diff
@@ -92,6 +92,13 @@ public function search(Search $search): Result
                     $this->escapeFilterValue($filter->longitude),
                     $this->escapeFilterValue($filter->distance),
                 ),
+                $filter instanceof Condition\GeoBoundingBoxCondition => $filters[] = \sprintf(
+                    '_geoBoundingBox([%s, %s], [%s, %s])',
+                    $this->escapeFilterValue($filter->minLatitude),
+                    $this->escapeFilterValue($filter->minLongitude),
+                    $this->escapeFilterValue($filter->maxLatitude),
+                    $this->escapeFilterValue($filter->maxLongitude),
```

Comment:
Meilisearch seems to be Top Right to Bottom Left while others are the other way around. I even think about make that default because JS libraries like Google Maps and Mapbox return NorthEast (TopRight) -> SouthWest (BottomLeft) as the bounded context for a map.